### PR TITLE
test(e2e-tests): feature file and steps for ACP integration

### PR DESCRIPTION
**Pull request by frank-dodion**
## Ticket Number
UCD-1352 and UCD-1353

## Description of change
Feature file and steps for ACP integration

### CODE REVIEW

File: e2e-tests/cypress/integration-with-acp/eligibility-check-decision-date.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@wip
+Feature: Check that decision date is valid and eligible to commence an appeal with ACP integration
+  Note - This feature is intended to support redirection to ACP service as part of initial MVP
+
+  Scenario: (1) Decision date less than 12 weeks old can proceed
+    Given the application decision date is requested
+    When a decision date is provided that is no more than 12 weeks old
+    Then the appeal can be commenced
+
+  Scenario: (2) Decision date older than 12 weeks cannot proceed
```
Comment (a6software) [inline e2e-tests/cypress/integration-with-acp/eligibility-check-decision-date.feature:10]: is it worth covering a decision date that is exactly 12 weeks old ?

File: e2e-tests/README.md [hunk 2]
```diff
@@ -41,3 +41,13 @@ for e2e tests in the pipelines. To create the required files locally run this co
 ````
 ./create-large-test-files.sh
 ````
+
+#### Integration with ACP
```
Comment (a6software) [inline e2e-tests/README.md:45]: `#### Integration with Appeals Casework Portal (ACP)`

i did not understand what ACP meant

File: e2e-tests/cypress/integration-with-acp/handle-unavailable-routes.feature [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      | "Appellant submission - Planning application - Application number"     |
+      | "Appellant submission - Planning application - Upload application"     |
+      | "Appellant submission - Planning application - Upload decision letter" |
+      | "Appellant submission - Your appeal - Appeal statement"                |
+      | "Appellant submission - Appeal site - Site location"                   |
+      | "Appellant submission - Appeal site - Site ownership"                  |
+      | "Appellant submission - Appeal site - Site access"                     |
+      | "Appellant submission - Appeal site - Site safety"                     |
+      | "Appellant submission - Appeal answers"                                |
+      | "Appeal submission"                                                    |
```
Comment (a6software) [inline e2e-tests/cypress/integration-with-acp/handle-unavailable-routes.feature:42]: worth covering the site root url here ?

File: e2e-tests/cypress/integration-with-acp/eligibility-check-decision-date.feature (hunk 1 above)
Comment (a6software) [reply e2e-tests/cypress/integration-with-acp/eligibility-check-decision-date.feature:10]: this appears covered with 84 days (?)

File: e2e-tests/README.md (hunk 2 above)
Comment (frank-dodion) [reply e2e-tests/README.md:45]: OK. I have updated the readme to include the full name and the url link.

File: e2e-tests/cypress/integration-with-acp/eligibility-check-decision-date.feature (hunk 1 above)
Comment (frank-dodion) [reply e2e-tests/cypress/integration-with-acp/eligibility-check-decision-date.feature:10]: Yes.

File: e2e-tests/cypress/integration-with-acp/handle-unavailable-routes.feature (hunk 3 above)
Comment (frank-dodion) [reply e2e-tests/cypress/integration-with-acp/handle-unavailable-routes.feature:42]: I don't think we have clear idea of the expected behaviour on landing at root url so I won't extend the tests here at this time.

**Review by Fenisay — APPROVED** [review]

**Pull request merged**